PostProcessing/Shaders/xRLib.hlsl

```diff
@@ -24,6 +24,8 @@ CBUFFER_START(UnityStereoEyeIndex)
 CBUFFER_END
 #endif
 
+float rvsGlobal;
```

Comment:
Can we name this uniform more explicitely? `rsvGlobal` is a bit vague...

Reply:
Yeah, I don't know why I used such a crappy name.  I'll use _RenderViewportScaleFactor